.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,228 @@
+name: Publish Docker Image
+
+on: [push, pull_request]
+#  push:
+#    branches:
+#      - main
+#    tags:
+#      - v1.*
+#
+env:
+  REPOSITORY: ${{ secrets.DOCKER_HUB_ORGS }}/fluentd-kubernetes-daemonset
+jobs:
+  amd64:
+    runs-on: ubuntu-latest
```

Comment:
Nitpick: If we use windows-latest here instead of ubuntu-latest, we can also publish Windows container images via GitHub Actions. Fluent Bit defines such workflows. See: https://github.com/fluent/fluent-bit/blob/master/.github/workflows/call-build-images.yaml#L283

Reply:
But, this work shouldn't be included in this PR. Because windows build takes a long time to create their images. :(

Reply:
Yes, it's a out of scope in this PR.